BusBoard.ConsoleApp/App.config

```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8" ?>
 <configuration>
     <startup> 
-        <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.5.2" />
+        <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.8" />
```

Comment:
I probably would have committed this file and the other config change together in a separate commit, so that it doesn't clutter the actually work that you have done

Reply:
Yeah makes sense